vscode/build.mjs

```diff
@@ -80,16 +90,20 @@ function buildBundle() {
   );
 }
 
-async function buildWatch() {
+export async function watchVsCode() {
```

Comment:
When I introduce errors in the VS Code files and save, I don't get the error output in the console. Would it be possible to surface them?

![image](https://github.com/microsoft/qsharp/assets/16928427/df78ca1e-8c5c-4c37-8b4e-cdb1036d7a28)

Reply:
That's because esbuild doesn't do TypeScript type checking. If you look at the npm build script, it does a 'npm run tsc:check' (which doesn't build anything, just checks types) followed by an esbuild (which doesn't type check anything, just bundles).

I can look into running both (not sure if 'tsc:check' can run in a separate 'watch' mode also). But your editor should have TypeScript errors as you make changes that introduce type errors, and this script is intended to be as you're editing/saving code, not run in a pipeline.

Reply:
I added watchers that do the TypeScript checking on all projects now. See if that works for you.